pyproject.toml

```
@@ -1,147 +1,134 @@
+[build-system]
+requires = [
+  "hatchling",
+  "hatch-vcs"
+]
+build-backend = "hatchling.build"
+
 [project]
-  name = "cosmology.compat.camb"
-  version = "0.1.0"
-  description = "Cosmology API standard compatability library for CAMB"
-  readme = "README.rst"
-  requires-python = ">=3.9"
-  license = {file = "LICENSE"}
-  keywords = ["cosmology", "API"]
-  authors = [
-    {name = "Nathaniel Starkman", email = "n.starkman@mail.utoronto.ca"},
-    {name = "Nicolas Tessore", email = "n.tessore@ucl.ac.uk"}
-  ]
-  maintainers = [
-    {name = "Nathaniel Starkman", email = "n.starkman@mail.utoronto.ca"},
-    {name = "Nicolas Tessore", email = "n.tessore@ucl.ac.uk"}
-  ]
-  classifiers = [
-    "Intended Audience :: Science/Research",
-    "Intended Audience :: Developers",
-    "License :: OSI Approved :: BSD License",
-    "Operating System :: OS Independent",
-    "Programming Language :: Python :: 3",
-  ]
-  dependencies = [
-    "cosmology.api",
-    "camb",
-    "numpy>=1.20",
-  ]
+name = "cosmology.compat.camb"
+description = "Cosmology API compatibility layer for CAMB"
+dynamic = [
+  "version"
+]
+readme = "README.rst"
+requires-python = ">=3.9"
+authors = [
+  {name = "Nathaniel Starkman", email = "n.starkman@mail.utoronto.ca"},
```

This email is going to go away in a year. New one is 
```suggestion
  {name = "Nathaniel Starkman", email = "starkman@mit.edu"},
```